examples/lines_bars_and_markers/linestyles.py

```diff
@@ -3,50 +3,65 @@
 Linestyles
 ==========
 
-This examples showcases different linestyles copying those of Tikz/PGF.
+Linestyle can be provided as simple as *solid* , *dotted*, *dashed*
```

Comment:
~~~
Simple linestyles can be defined using the strings "solid", "dotted", "dashed"
or "dashdot". More refined control can be achieved by providing a dash tuple
``(offset, (on_off_seq))``. For example, ``(0, (3, 10, 1, 15))`` means
3pt-line, 10pt-space, 1pt-line, 15pt-space with no offset. See also
`.Line2D.set_linestyle`.
~~~

Reply:
If you are going to change this line, please remove the dashes between 'pt' and 'line'/'space'; they are not technically required.